src/lib/eZ/RichText/Resources/schemas/docbook/docbook.rng

```diff
@@ -1120,7 +1120,10 @@
         <a:documentation>The text of the title of a section of a document or of a formal block-level element</a:documentation>
         <ref name="db.title.attlist"/>
         <zeroOrMore>
-          <ref name="db.all.inlines"/>
+          <choice>
+            <ref name="db.all.inlines"/>
+            <ref name="db.literallayout"/>
+          </choice>
```

Comment:
Asking this is thread so we can close it if irrelevant, but @alongosz do you remember if we have adaptions on this schema and are diverging from docbook spec on other things?

Either way, does:
- newer versions of docbook fix this? 
- or are there other ways we could deal with new lines in headers?

Another question: We might need to adapt ezplatform-xmltext-fieldtpye migratiomn for this or? (line breaks where supported in headers there too)

Reply:
> Another question: We might need to adapt ezplatform-xmltext-fieldtype migratiomn for this or? (line breaks where supported in headers there too)

Hmm, but did it work there then? Didn't we use there the same RelaxNG files? It should fail the same way. Or was XmlText spec different one? I'm sorry I don't remember this, hard to keep track of things, especially related to eZ Publish.

Reply:
Unsure, maybe @vidarl remembers

Reply:
According to the latest DocBook (5.2) documentation `literallayout  is still not mentioned among possible children for `title`, ref: https://tdg.docbook.org/tdg/5.2/title.html. I reverted customizing DocBook schema in `docbook.rng` and moved it to `ezpublish.rng` in 07fa7018c70bab79f11522b28e010cd12f845c43. As agreed, `literallayout` is an inline element, so I just override `db.all.inlines` section to take it into account.